lib/components/artist/related_artists/index.js

```diff
@@ -3,35 +3,67 @@
 
 import Relay from 'react-relay';
 import React from 'react';
-import { ScrollView, StyleSheet, View } from 'react-native';
+import { ScrollView, StyleSheet, View, Dimensions } from 'react-native';
 
 import colors from '../../../../data/colors';
 import SerifText from '../../text/serif';
 import ImageView from '../../opaque_image_view';
 import RelatedArtist from './related_artist'
 
 class RelatedArtists extends React.Component {
+    constructor(props) {
+    super(props);
+    this.state = { columns: this.columnCount()};
+    this.onLayout = this.onLayout.bind(this);
+  }
+  
+  columnCount() {
+    const isPhone = Dimensions.get('window').width < 700;
+    const isPadHorizontal = Dimensions.get('window').width > 1000;
```

Comment:
Maybe reduce method calls a bit more by storing the `window` object?

(Not sure about this case, but might be relevant in cases where behind the scenes it calls a native module and it needs to go over the bridge back and forth multiple times.)